stackslib/src/net/mod.rs

```diff
@@ -1371,12 +1384,15 @@ pub struct NetworkResult {
     pub unhandled_messages: HashMap<NeighborKey, Vec<StacksMessage>>,
     pub blocks: Vec<(ConsensusHash, StacksBlock, u64)>, // blocks we downloaded, and time taken
     pub confirmed_microblocks: Vec<(ConsensusHash, Vec<StacksMicroblock>, u64)>, // confiremd microblocks we downloaded, and time taken
+    pub nakamoto_blocks: Vec<(ConsensusHash, NakamotoBlock, u64)>, // Nakamoto blocks we downloaded, and time taken
```

Comment:
The `ConsensusHash` field is unnecessary, since it's already present in the `NakamotoBlock` header.